src/envoy/mixer/mixer_control.cc

```diff
@@ -220,199 +35,50 @@ class EnvoyTimer : public ::istio::mixer_client::Timer {
   Event::TimerPtr timer_;
 };
 
-}  // namespace
+// Create all environment functions.
```

Comment:
It doesn't have to be addressed in this PR, but can we reconcile use of `platform` and `environment` between istio/proxy and istio/mixerclient?

Reply:
Mixerclient is using Environment,  I will try to replace all "platform" with environment

Reply:
Hmm,  "platform" is not used.

Reply:
See https://github.com/istio/mixerclient/pull/248/files#diff-52908b68b42c30e44f87d00288171c3fR50

Reply:
I missed that one.  Will change it in MixerClient